### DISCUSSION TITLE
WebUI unusable after some hours

### DISCUSSION THREAD

**Discussion by anzimber** [jomjol/AI-on-the-edge-device#2430] [General] [answered]
### The Problem

Hi,

not sure how to describe the issue in the best way.
Installtion was working fine. Setup of recognition numbers also. But as soon as I try to reconnect again to WebUI after some hours the UI is not working expected. Takes more then 1minute to open Startpage. Moving in the page also is extremly slow. Loading of logs never works(only in the beginning after the reboot).

Pls tell me what and how to resume to support to fix the issue.

br and thanks
ori

### Version

Release: v15.2.4 (Commit: cbd14a2)

### Logfile

```shell
<pre>



        _
    .__(.)< ( oh oh! This page does not exist! )
    \___)


                You could try your <a href=index.html target=_parent>luck</a> here!</pre>
<script>document.cookie = "page=overview.html"</script>
```

### Expected Behavior

WebUI is usable and loading reasonable fast.

**Comment by anzimber** [comment]
additionally if I open log file via "file server"...

```
[4d14h39m00s] 2023-05-22T08:36:51	<ERR>	[OTA FILE] Failed to read file: /sdcard/log/message/log_2023-05-22.txt!
````

**Comment by caco3** [comment]
How good is your WIFI connection? Whats the RSSI?
Try to put it near your accesspoint and check it works better there.
Slow Web UI can also be due to a corrupted or non-matching SD-Card.

Please provide a full DEBUG log.

**Reply by anzimber** [reply]
Hi @caco3,

sorry, for late reply.

The RSSI values are in range -40dbm +-2 and my esp32 is 1,5m next to my wlan ap.

when I download the files via wget from fileserver it ends up with http errors...


so files in attachment are the best versions I could hand over to you :-/ wget performance is around 2-3kB/s max... :/
[log_2023-05-22.txt](https://github.com/jomjol/AI-on-the-edge-device/files/11551578/log_2023-05-22.txt)
[log_2023-05-23.txt](https://github.com/jomjol/AI-on-the-edge-device/files/11551580/log_2023-05-23.txt)
[log_2023-05-24.txt](https://github.com/jomjol/AI-on-the-edge-device/files/11551626/log_2023-05-24.txt)



hope that helps ...

btw ... with debugging set at least I could always open the "log viewer" ....

br
ori

**Comment by caco3** [comment]
> <ERR>	[OTA FILE] File does not exist: /System%20Volume%20Information

This is simply a 404 error because you somehow try to access an URL that clearly does not exist on the device.

A RSSI of -40dbm is great and should be ok. But it doesn't guarantee that the connection can provide a high throughput. You could try with an external antenna if not using yet. 

Maybe you could try with a different SD-Card and a different ESP board.

There where other people who had slow UI. In most cases they used ESP Chip revision 3.1, but you have 3.0 (`<INF>	[MAIN] Device info: CPU cores: 2, Chip revision: 300`), so that should not be the issue (see https://github.com/jomjol/AI-on-the-edge-device/discussions/2405).

**Reply by caco3** [reply]
BTW, even with a weak RSSI (-85dBm), I can reach >20..30 kB/s with wget!

**Comment by friedpa** [comment]
It all depends on the latency. Just ping the device. If you are over 75-100 msec -> "Houston we have a problem" :o)

**Reply by anzimber** [reply]
and what can I do against that ???

:)

10.0.0.12 is ESP32 and my host is directly connected via 1GE LAN to my fritzbox
```
 ping 10.0.0.12
PING 10.0.0.12 (10.0.0.12) 56(84) bytes of data.
64 bytes from 10.0.0.12: icmp_seq=1 ttl=255 time=55.3 ms
64 bytes from 10.0.0.12: icmp_seq=2 ttl=255 time=115 ms
64 bytes from 10.0.0.12: icmp_seq=3 ttl=255 time=190 ms
64 bytes from 10.0.0.12: icmp_seq=4 ttl=255 time=157 ms
64 bytes from 10.0.0.12: icmp_seq=5 ttl=255 time=67.9 ms
64 bytes from 10.0.0.12: icmp_seq=6 ttl=255 time=68.9 ms
64 bytes from 10.0.0.12: icmp_seq=7 ttl=255 time=143 ms
64 bytes from 10.0.0.12: icmp_seq=8 ttl=255 time=113 ms
64 bytes from 10.0.0.12: icmp_seq=9 ttl=255 time=33.0 ms
64 bytes from 10.0.0.12: icmp_seq=10 ttl=255 time=674 ms
64 bytes from 10.0.0.12: icmp_seq=11 ttl=255 time=1001 ms
64 bytes from 10.0.0.12: icmp_seq=12 ttl=255 time=614 ms
64 bytes from 10.0.0.12: icmp_seq=13 ttl=255 time=201 ms
64 bytes from 10.0.0.12: icmp_seq=14 ttl=255 time=447 ms
64 bytes from 10.0.0.12: icmp_seq=15 ttl=255 time=349 ms
64 bytes from 10.0.0.12: icmp_seq=16 ttl=255 time=364 ms
64 bytes from 10.0.0.12: icmp_seq=17 ttl=255 time=101 ms
64 bytes from 10.0.0.12: icmp_seq=18 ttl=255 time=77.8 ms
64 bytes from 10.0.0.12: icmp_seq=19 ttl=255 time=58.5 ms
64 bytes from 10.0.0.12: icmp_seq=20 ttl=255 time=73.9 ms
64 bytes from 10.0.0.12: icmp_seq=21 ttl=255 time=206 ms
64 bytes from 10.0.0.12: icmp_seq=22 ttl=255 time=126 ms
64 bytes from 10.0.0.12: icmp_seq=23 ttl=255 time=201 ms
64 bytes from 10.0.0.12: icmp_seq=24 ttl=255 time=91.5 ms
64 bytes from 10.0.0.12: icmp_seq=25 ttl=255 time=193 ms
64 bytes from 10.0.0.12: icmp_seq=26 ttl=255 time=203 ms
64 bytes from 10.0.0.12: icmp_seq=27 ttl=255 time=68.3 ms
64 bytes from 10.0.0.12: icmp_seq=28 ttl=255 time=271 ms
^C
--- 10.0.0.12 ping statistics ---
29 packets transmitted, 28 received, 3.44828% packet loss, time 28034ms
rtt min/avg/max/mdev = 32.973/223.801/1001.415/218.456 ms
```


first thing I'll do is completely reinstall it these evening with latest rolling changes ...

**Reply by caco3** [reply]
> first thing I'll do is completely reinstall it these evening with latest rolling changes ...

That will not fix it.

if you have 5 GHz wifi running on your AP, disable it for a test. also use a lower 2.4 GHz Channel. Search the discussions for details.

**Reply by anzimber** [reply, marked as answer]
it was wireless channel 2.. changed to 9 and now it is working as expected....

**Comment by friedpa** [comment]
Did you change the Ethernet Cable to check if the cable has maybe a problem (not soldered correctly, therefore interference reflections)?

**Reply by anzimber** [reply]
my ethernet is fine... no problems with that ....
